recipes-wigwag/deviceOSWD/deviceoswd_0.0.2.bb

```diff
@@ -35,7 +35,7 @@ do_compile() {
     cd ${S}
 
     cd ${S}/deps
-    TOOLCHAIN=`echo $AR | sed 's/.\{3\}$//'` ./install-deps.sh
+    TOOLCHAIN=`echo $AR | sed 's/.\{7\}$//'` ./install-deps.sh
```

Comment:
Why did the positional results change?

Reply:
`$AR` used to be `arm-poky-linux-gnueabi-ar` and now it's `arm-poky-linux-gnueabi-gccXXX`

Reply:
1) lets merge this with dev instead of master
2) lets figure out how to make this simultaneously thud and warrior compatible since its the only difference.  Either a better way to detect the AR or a if/else against warrior / thud.